src/components/FormSelectList.vue

```diff
@@ -247,8 +261,15 @@
         };
       },
       valueProxy: {
-        get() { return this.value; },
-        set(val) { return this.$emit('input', val); }
+        get() {
+          if (this.options.renderAs === "dropdown") {
+            return this.arrayHasSelectedOptions(this.value) ? this.value : [];
```

Comment:
What happend if the `SelectList` is a **Single Value** or **Single Object** and the `this.value` is (incorrectly) an array?
- in this case `arrayHasSelectedOptions` `returns true` and valueProxy will return `[]`, is that correct?
-

Reply:
As you said, if it is a single value/object, arrayHasSelectedOptions return true and this.arrayHasSelectedOptions(this.value) ? this.value : [] ==> this.value as it was before the fix
The fix modifies the value just when the SelectList is multiple.